packages/iris-grid/src/sidebar/aggregations/Aggregations.tsx

```diff
@@ -40,6 +40,7 @@ export type AggregationSettings = {
 
 export type AggregationsProps = {
   isRollup: boolean;
+  isEditable: boolean;
```

Comment:
Prop name should probably be more specific - as you can still edit the aggregations, you just can't edit the position.

```suggestion
  isPositionEditable: boolean;
```

Or even having something like `availablePositions` and passing in `top` or `bottom` and just disabling the ones that are not available, as even `isPositionEditable` does not convey that it will be pinned to the top.